Week3/homework/step2-2.js

```diff
@@ -10,9 +10,6 @@ function threeFive(startIndex, stopIndex, threeCallback, fiveCallback) {
       threeCallback(elem);
     } else if (elem % 5 === 0) {
```

Comment:
Almost there, with this `else` statement it's stopping right away once it has the `if` condition true, the this else never reached out!
```suggestion
     }
    if (elem % 5 === 0) {
```

Reply:
Thanks @Meno-101, that was useful :+1: